src/components/InputBar.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './InputBar.css';
+const InputBar = () => {
+  const [search, setSearch] = useState('');
+
+  const handleChange = e => {
+    setSearch(e.target.value);
+  };
+  return (
+    <div className="InputBar">
```

Comment:
추후에 엔터를 클릭하거나 버튼을 클릭할 때 검색 기능이 실행되기 위해서 form 태그로 사용해주면 좋을 것 같아요~
```suggestion
    <form className="InputBar">
```